app/_locales/fr/messages.json

```diff
@@ -457,44 +818,176 @@
   "refundAddress": {
     "message": "Votre adresse de remboursement"
   },
+  "reject": {
+    "message": "Rejeter"
+  },
+  "rejectAll": {
+    "message": "Tout rejeter"
+  },
+  "rejectTxsN": {
+    "message": "Rejeter les transactions $1"
+  },
+  "rejectTxsDescription": {
+    "message": "Vous êtes sur le point de rejetter en groupe les transactions $1."
+  },
   "rejected": {
     "message": "Rejeté"
   },
+  "reset": {
+    "message": "Reinitialiser"
+  },
+  "resetAccount": {
+    "message": "Reinitialiser le compte"
+  },
+  "resetAccountDescription": {
+    "message": "Reinitialiser votre compte va effacer votre historique de transactions."
+  },
+  "restoreFromSeed": {
+    "message": "Restaurer le compte ?"
+  },
+  "restoreVault": {
+    "message": "Restaurer le Coffre"
+  },
+  "restoreAccountWithSeed": {
+    "message": "Restaurer votre compte avec une phrase Seed."
+  },
   "required": {
     "message": "Obligatoire"
   },
   "retryWithMoreGas": {
-    "message": "Réessayez avec un prix plus élevé du gaz ici"
+    "message": "Réessayer avec un prix de gaz plus élevé"
+  },
+  "walletSeed": {
+    "message": "Seed du portefeuille"
+  },
+  "restore": {
+    "message": "Restaurer"
+  },
+  "revealSeedWords": {
+    "message": "Révéler les mots Seed"
+  },
+  "revealSeedWordsTitle": {
+    "message": "Phrase Seed"
+  },
+  "revealSeedWordsDescription": {
+    "message": "Si jamais vous changez de navigateur ou d'ordinateur, vos aurez besoin de cette phrase seed pour accéder à vos comptes. Sauvegardez la quelque part de sûr et secret."
+  },
+  "revealSeedWordsWarningTitle": {
+    "message": "Ne communiquez PAS cette phrase à quelqu'un !"
```

Comment:
Can we remove the space before the exclamation mark?

Reply:
same as above